openedx/core/djangoapps/waffle_utils/tests/test_init.py

```diff
@@ -47,7 +46,7 @@ def setUp(self):
         RequestCache.clear_all_namespaces()
 
     @override_settings(WAFFLE_FLAG_CUSTOM_ATTRIBUTES=[NAMESPACED_FLAG_NAME])
-    @patch('openedx.core.djangoapps.waffle_utils.set_custom_attribute')
+    @patch.object(edx_toggles.toggles.internal.waffle, 'set_custom_attribute')
```

Comment:
Can you add a TODO comment acknowledging that we shouldn't be referring to `internal`?  I know we discussed it, but please ensure we have a task for this cleanup.

Reply:
Ok, I'll add a TODO at the top-level import.